src/core/client/admin/components/UserRole/SiteModeratorModal.tsx

```diff
@@ -62,11 +68,11 @@ const SiteModeratorModal: FunctionComponent<Props> = ({
           <Flex justifyContent="flex-end">
             <CardCloseButton onClick={onCancel} ref={firstFocusableRef} />
           </Flex>
-          <Form
+          <Formik
             onSubmit={onSubmit}
             initialValues={{ siteIDs: selectedSiteIDs }}
           >
-            {({ handleSubmit, submitError, submitting }) => (
+            {({ handleSubmit, status, isSubmitting }) => (
```

Comment:
Formik's API matches final form's use of child render props, but context can be used as an alternative as seen in the signupWithEmail form